frontend/src/v5/ui/routes/dashboard/projects/drawings/uploadDrawingRevisionForm/uploadList/uploadList.component.tsx

```diff
@@ -0,0 +1,100 @@
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { UploadItemFields } from '@/v5/store/drawings/drawingRevisions/drawingRevisions.types';
+import { useContext } from 'react';
+import { SortingDirection } from '@components/dashboard/dashboardList/dashboardList.types';
+import { useOrderedList } from '@components/dashboard/dashboardList/useOrderedList';
+import { UploadFilesContext } from '@components/shared/uploadFiles/uploadFilesContext';
+import { DashboardListHeader, DashboardListHeaderLabel as Label } from '@components/dashboard/dashboardList';
+import { FormattedMessage } from 'react-intl';
+import { UploadListItem } from './uploadListItem/uploadListItem.component';
+import { ListContainer } from './uploadList.styles';
+import { UploadListItemRowWrapper } from './uploadListItem/uploadListItem.styles';
+
+type IUploadList = {
+	values: UploadItemFields[];
+	removeUploadById: (id: string) => void;
+	isUploading: boolean;
+};
+
+const DEFAULT_SORT_CONFIG = {
+	column: ['file'],
+	direction: [SortingDirection.DESCENDING, SortingDirection.ASCENDING],
+};
+
+export const UploadList = ({
+	values,
+	removeUploadById,
+	isUploading,
+}: IUploadList): JSX.Element => {
+	const { selectedId, setSelectedId } = useContext(UploadFilesContext);
+	const { sortedList, setSortConfig } = useOrderedList(values, DEFAULT_SORT_CONFIG);
+
+	const deleteItem = (id, isSelected) => {
+		removeUploadById(id);
+		if (isSelected) setSelectedId();
+	};
+
+	return (
+		<>
+			<DashboardListHeader onSortingChange={setSortConfig} defaultSortConfig={DEFAULT_SORT_CONFIG}>
+				<Label name="filename" minWidth={122} sort>
+					<FormattedMessage id="drawing.uploads.list.header.filename" defaultMessage="Filename" />
+				</Label>
+				<Label name="destination" width={352}>
+					<FormattedMessage id="drawing.uploads.list.header.destination" defaultMessage="Drawing Destination" />
+				</Label>
+				<Label name="statusCode" width={352}>
+					<FormattedMessage id="drawing.uploads.list.header.statusCode" defaultMessage="Status Code" />
+				</Label>
+				<Label name="revisionCode" width={isUploading ? 359 : 399}>
+					<FormattedMessage id="drawing.uploads.list.header.revisionName" defaultMessage="Revision Code" />*
+				</Label>
+				<Label name="description" width={297} hidden={!isUploading}>
+					<FormattedMessage id="drawing.uploads.list.header.description" defaultMessage="Description" />
```

Comment:
Should this be description? I see it in the mockup, but that seems to be refering to a different input which I'm guessing has been removed?